portfolio/src/main/webapp/script.js

```diff
@@ -27,8 +27,9 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
-function getData() {
-  fetch('/data').then(response => response.text()).then((quote) => {
-    document.getElementById('data-container').innerText = quote;
-  });
+async function getData() {
+ fetch('/data').then(response => response.json()).then((data) => {
+      console.log(data);
```

Comment:
What do you think about removing this debug statement?

Reply:
My newest branch kind of addressed this issue, should I just close this pull request or would it be better to also commit the change here?

Reply:
I think you should merge this PR and resolve conflicts if they exist when you merge the next PR.